tensorflow/compiler/mlir/tosa/transforms/strip_quant_types.cc

```diff
@@ -1,27 +1,26 @@
-/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-==============================================================================*/
-
-// This pass converts a TFLite uint8 graph to the int8 domain, with adaptors at
-// input and output tensors. This is needed because TOSA precision is
-// implemented in the int8 domain. This pass does:
-// 1. match TFL::QConst with uint8, generate TFL::QConst with int8 with value
-// remapped.
-// 2. insert tosa.RESCALE uint8 -> int8 if block argument (placeholder of graph)
-// is uint8 typed.
-// 3. insert tosa.RESCALE int8 -> uint8 if original returned tensor is uint8
-// typed.
+// Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ==============================================================================*/
+
+// This pass removes the quantized type from output Tensors attached to TOSA Ops
```

Comment:
This pass needs to maintain stripping quantized information from the entire model, including function arguments.